### DISCUSSION TITLE
Is it possible to change the theme style at runtime.

### DISCUSSION THREAD

**Discussion by heng30** [slint-ui/slint#5213] [Q&A]
I know it can change the style follow the device theme. However, I want to click a button to change the style from material-light to material-dark. Do not follow the device theme. Is it possible?

**Comment by ogoffart** [comment]
It is not possible to change the style at runtime.
But from Slint 1.6 (the master branch) is it possible to change the color scheme with Palette global. https://snapshots.slint.dev/master/docs/slint/src/language/builtins/globals.html#palette (`color-scheme` property)
The Slint gallery does that https://github.com/slint-ui/slint/blob/25ef8f8711c048874abdd9dfd934432ee1545c1c/examples/gallery/ui/pages/page.slint#L34

**Reply by heng30** [reply]
Thanks, It works for me. I will try it after v1.6 published to crates.io
